en/firefox_privacy_notice.md

```diff
@@ -83,64 +89,37 @@ Read the full documentation [here](https://firefox-source-docs.mozilla.org/toolk
+### Mozilla Accounts {: #Mozilla-accounts }
 
-* __Location data__: For security purposes, we store the IP addresses used to access your Firefox Account in order to approximate your city and country. We use this data to send you email alerts if we detect suspicious activity, such as account logins from other locations.
+Mozilla receives registration, location, interaction and technical data when you create and use a Mozilla account. See the [Mozilla Accounts Privacy Notice](https://www.mozilla.org/privacy/mozilla-accounts/) for more information. 
 
-* __Interaction data__: We receive data such as your visits to the Firefox Accounts website, dashboards and menu preferences, what products and services you use in connection with your Firefox Account, and your interactions with our emails and SMS messages. We use this to understand your use of our products and services and to send you more useful Firefox Account Tips and in-product messages. Learn more about interaction data we collect in the [Glean Dictionary](https://dictionary.telemetry.mozilla.org/apps/accounts_frontend).
+### Review Checker {: #review-checker }
 
-* __Technical data__: To display which devices are synced to your Firefox Account and for security functionality, we store your device operating system, browser and version, timestamp, locale, and the same information for devices connected to your account. If you use your Firefox Account to log into other websites or services (such as AMO or Pocket), we receive the timestamp of those log-ins.
+Users in the US can opt-in to Review Checker, our guide to product reviews powered by Mozilla’s [Fakespot](https://www.fakespot.com/), which helps you understand how reliable product reviews are on [certain shopping sites](https://support.mozilla.org/en-US/kb/review-checker-review-quality). 
 
-Read the full documentation or learn more, including how to manage your Firefox Account data or our data practices for [websites and email](https://www.mozilla.org/privacy/websites/).  You can also read the privacy notices for our Firefox Account connected services, which are:
-
-* [Firefox Monitor](https://www.mozilla.org/privacy/firefox-monitor)
-* [Firefox Notes](https://addons.mozilla.org/firefox/addon/notes-by-firefox/)
-* [Pocket](https://getpocket.com/privacy/)
-* [Firefox Sync](https://www.mozilla.org/privacy/firefox/#sync)
-* [Firefox Relay](https://www.mozilla.org/privacy/firefox-relay/)
-* [Mozilla VPN](https://www.mozilla.org/privacy/mozilla-vpn/)
+* __Technical and interaction data__: We use Oblivious HTTP ([OHTTP](https://datatracker.ietf.org/doc/draft-ietf-ohai-ohttp/)) with our partner [Fastly](https://www.fastly.com/privacy/) to provide the service, while ensuring that any data identifying products, URLs, and request payloads are unlinkable to you. For more information about Review Checker, including what websiteshow to turn it off, you can read the [documentation](https://support.mozilla.org/kb/review-checker-review-quality). 
```

Comment:
I don't know what you mean with `including what websiteshow to turn it off`, it looks like it was originally 2 separate sentences?